salt/kube-apiserver/init.sls

```diff
@@ -12,6 +12,24 @@ include:
          cn = grains['nodename'],
          o = pillar['certificate_information']['subject_properties']['O']) }}
 
+/etc/pki/kube-apiserver-proxy-client.crt:
+  file.managed:
+    - contents_pillar: api:admission_webhook:cert
```

Comment:
This cert/keypair is used for more than just the admission webhook. We shouldn't call the pillar this :)